github-actions/bazel/setup/action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Setup Bazel'
+description: 'Setup and configure bazel for CI use'
+author: 'Angular'
+
+inputs:
+  bazelrc:
+    default: .bazelrc.user
+    description: |
+      The `bazelrc` file location to add the `repository_cache` flag to. Defaults
+      to .bazelrc.user
+
+runs:
+  using: composite
+  steps:
+    - name: Configure action caching for bazel version downloaded by bazelisk
+      uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # tag=v3.3.1
+      with:
+        path: |
+          ~/.cache/bazelisk
+          \%LocalAppData\%/bazelisk
+        key: bazel-version-${{ hashFiles('**/.bazelversion') }}
+        restore-keys: |
+          bazel-version-
```

Comment:
We should not have a restore key suffix here. If the bazel version changes, it's better to not re-use any cache so that we do not grow indefinitely with more Bazel versions

Reply:
Ahh good call, didn't think about that one.